lib/versioning/rez/index.ts

```diff
@@ -57,3 +67,317 @@
+  }
+  if (lowerBound.test(currentValue)) {
+    if (currentValue.includes('+')) {
+      return npm2rezplus(pep440Value);
+    }
+    return pep440Value;
+  }
+  if (upperBound.test(currentValue)) {
+    return pep440Value;
+  }
+  if (ascendingRange.test(currentValue)) {
+    // Replace version numbers but keep rez format, otherwise we just end up trying
+    // to convert every single case separately.
+    const match = ascendingRange.exec(currentValue);
+    const lowerBoundAscCurrent = match.groups.range_lower_asc;
+    const upperBoundAscCurrent = match.groups.range_upper_asc;
+    const lowerAscVersionCurrent = match.groups.range_lower_asc_version;
+    const upperAscVersionCurrent = match.groups.range_upper_asc_version;
+    const [lowerBoundAscPep440, upperBoundAscPep440] = pep440Value.split(', ');
+    const lowerAscVersionNew = new RegExp(versionGroup).exec(
+      lowerBoundAscPep440
+    )[0];
+    const upperAscVersionNew = new RegExp(versionGroup).exec(
+      upperBoundAscPep440
+    )[0];
+    const lowerBoundAscNew = lowerBoundAscCurrent.replace(
+      lowerAscVersionCurrent,
+      lowerAscVersionNew
+    );
+    const upperBoundAscNew = upperBoundAscCurrent.replace(
+      upperAscVersionCurrent,
+      upperAscVersionNew
+    );
+    const separator = currentValue.includes(',') ? ',' : '';
+
+    return lowerBoundAscNew + separator + upperBoundAscNew;
+  }
+  if (descendingRange.test(currentValue)) {
+    // Replace version numbers but keep rez format, otherwise we just end up trying
+    // to convert every single case separately.
+    const match = descendingRange.exec(currentValue);
+    const upperBoundDescCurrent = match.groups.range_upper_desc;
+    const lowerBoundDescCurrent = match.groups.range_lower_desc;
+    const upperDescVersionCurrent = match.groups.range_upper_desc_version;
+    const lowerDescVersionCurrent = match.groups.range_lower_desc_version;
+    const [lowerBoundDescPep440, upperBoundDescPep440] =
+      pep440Value.split(', ');
+
+    const upperDescVersionNew = new RegExp(versionGroup).exec(
+      upperBoundDescPep440
+    )[0];
+    const lowerDescVersionNew = new RegExp(versionGroup).exec(
+      lowerBoundDescPep440
+    )[0];
+    const upperBoundDescNew = upperBoundDescCurrent.replace(
+      upperDescVersionCurrent,
+      upperDescVersionNew
+    );
+    const lowerBoundDescNew = lowerBoundDescCurrent.replace(
+      lowerDescVersionCurrent,
+      lowerDescVersionNew
+    );
+    // Descending ranges are only supported with a comma.
+    const separator = ',';
+
+    return upperBoundDescNew + separator + lowerBoundDescNew;
+  }
+  return null;
+}
+
+export const api: VersioningApi = {
+  equals,
+  getMajor,
+  getMinor,
+  getPatch,
+  getNewValue,
+  getSatisfyingVersion,
+  isCompatible: isVersion,
+  isGreaterThan,
+  isLessThanRange,
+  isSingleVersion,
+  isStable,
+  isValid,
+  isVersion,
+  matches,
+  minSatisfyingVersion,
+  sortVersions,
+};
+export default api;
```

Comment:
Fix newline at end of file

```suggestion
export default api;

```